metadata-editor/app/controllers/EditsController.scala

```diff
@@ -212,6 +213,19 @@ class EditsController(
     (Json.toJson(metadata).as[JsObject]).as[Map[String, JsValue]]
   }
 
+  def softDelete(id: String) = auth.async { implicit req =>
+    val result = for {
+      _ <- editsStore.stringSet(id, Edits.DeletedAt, JsString(DateTime.now.toString))
```

Comment:
it looks like we're discarding the output of this command and mapping on the result of setting the deletedBy only, is there a way we could combine them into a single command to set both fields?

Reply:
yes we can use `jsonAdd` to add both
will update the PR now